src/styles/_common.scss

```diff
@@ -1,62 +1,60 @@
-#body {
```

Comment:
What's the reason for removing this body tag?

Reply:
It was removed by mistake.. Fixed